### PR TITLE
fix: oracle events not loading

### LINKED ISSUE #343: Oracle events malformed URI
### Context information

application version: 
device: 
browser: 
operating system: 

### Steps to reproduce
- Open http://localhost:8080/oracles/ok_2fVpBb8j58mNQcVjY7bwUVpBWZv6AySGDjqepNofU8XGGmEN53
- Check Chrome console

### What is expected?
- Oracle events are visible

### What is actually happening?

image

### PR DESCRIPTION
**Pull request by michele-franchi**
## Description

Fixes #343.

## Demo

image

## Checklist:

- [ ] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by michele-franchi** [comment]
@janmichek I had to remove the data `data-failed-state` component because for some reason the `v-if` do not make pagination work correctly. I will open a followup for this.

File: src/utils/format.js [hunk 1]
```diff
@@ -77,8 +77,7 @@ export function formatNullable(value) {
 }
 
 export function formatDecodeBase64(base64String) {
-  const decodedString = process.client ? window.atob(base64String) : Buffer.from(base64String, 'base64').toString('utf8')
-  return decodeURIComponent(escape(decodedString))
+  return process.client ? window.atob(base64String) : Buffer.from(base64String, 'base64').toString('utf8')
```
Comment (michele-franchi) [inline src/utils/format.js:80]: I checked other parts where this function is used and everything seems to be working fine.

**Review by janmichek — APPROVED** [review]
Ok cool, seems to be fixed. Please attach the the followup issue

**Comment by michele-franchi** [comment]
Followup https://github.com/aeternity/aescan/issues/345

**Pull request merged**